app/controllers/application_controller.rb

```diff
@@ -1,5 +1,32 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  helper_method :current_user
+
+  def current_user
+    User.find_by(id: session[:user_id])
+  end
```

Comment:
Probably worth putting if session[:user_id] at the end of this line to avoid querying the DB on every request even when you know there's no user id in the session.